themes/socialbase/templates/profile/profile--profile--compact_teaser.html.twig

```diff
@@ -31,9 +36,12 @@
       <a href="{{ profile_home }}" title="{{ 'View Profile'|t }}">{{ profile_name }}</a>
 
       {% if content.field_profile_function|render and content.field_profile_organization|render %} <br /> {% endif %}
+
       {{content.field_profile_function}}
 
-      {% if content.field_profile_function|render and content.field_profile_organization|render %} &bullet; {% endif %}
+      {% if not style == 'sky' %}
+        {% if content.field_profile_function|render and content.field_profile_organization|render %} &bullet; {% endif %}
```

Comment:
Perhaps we can move this HTML to a `.profile-function ~ .profile-organization:before { content: '\cdot'; }` style line in socialbase. That way the SocialBlue sky CSS can simply remove it?